Sync/DAO/Value/ReferenceValueDAO.php

```diff
@@ -18,20 +18,40 @@ class ReferenceValueDAO
     private $value;
 
     /**
-     * @return int
+     * @var string
```

Comment:
```suggestion
     * @var string|null
```
The other property can be null as well.